luda-editor/luda-editor-client/src/app/sequence_list/ops/sync_sequences_from_google_spreadsheet.rs

```diff
@@ -54,8 +55,11 @@ impl SequenceList {
 
 async fn sync_sequences_with_sheets(
     socket: &Socket,
-) -> Result<BTreeMap<String, Arc<Sequence>>, String> {
+) -> Result<LinkedHashMap<String, Arc<Sequence>>, String> {
     let sheets = google_spreadsheet::get_sheets().await?;
+    SequenceIndex::new(sheets.iter().map(|sheet| sheet.title.clone()).collect())
+        .save(socket)
+        .await?;
```

Comment:
Save spreadsheet's sequence order as `sequence_index.json` when we sync to spreadsheet

Reply:
Please make a simple function like 61~67 and just call it. Then you don't have to tell me what this code do.